tests/Unit/Css/CssDocumentTest.php

```diff
@@ -380,7 +375,7 @@ public function rendersValidNonConditionalAtRule(string $atRuleCss, string $cssB
     }
 
     /**
-     * @return array<non-empty-string, list<non-empty-string>>
+     * @return array<non-empty-string, array{0: non-empty-string}>
```

Comment:
This can provide one or two non-empty strings.  I guess there's a syntax like `?1:` for the optional element but don't know offhand.